### PR TITLE
Correction for the Cloud SOAR webhooks article

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request updates this section:
https://help.sumologic.com/docs/cloud-soar/automation/#configure-a-webhook-for-cloud-soar

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

PR created at the request of @alesscorona. It fixes an error I made in my review of his PR #3981.

### CODE REVIEW

**Comment by jpipkin1** [comment]
@alesscorona, could you review this PR?

File: docs/cloud-soar/automation.md [hunk 1]
```diff
@@ -476,7 +476,7 @@ You can configure a [webhook connection](/docs/alerts/webhook-connections/cloud-
 1. Click **+** and choose **Cloud SOAR** as the connection type. The **Create Cloud SOAR Connection** dialog is displayed.<br/><img src={useBaseUrl('img/cloud-soar/CSOAR-connection1.png')} alt="New connection" style={{border: '1px solid black'}} width="600"/> 
 1. Enter a **Name** and give an optional **Description** to the connection.
 1. The **URL** field shows your [Sumo Logic API endpoint](/docs/api/getting-started#sumo-logic-endpoints-by-deployment-and-firewall-security) followed by `/csoar/v3/incidents/`. For example, `https://api.us2.sumologic.com/api/csoar/v3/incidents/`
-1. The **Authorization Header** field is automatically populated with your [Basic Access](/docs/api/getting-started#basic-access-base64-encoded) authentication.
+1. In **Authorization Header**, enter your basic authentication access information for the header. For example, `Basic <base64 encode <accessId>:<accessKey>>`. For more information, see [Basic Access (Base64 encoded)](/docs/api/getting-started#basic-access-base64-encoded).
```
Comment (kimsauce) [inline docs/cloud-soar/automation.md:479]:
```suggestion
1. In **Authorization Header**, enter your basic authentication access information for the header. For example, `Basic <base64 encode <accessId>:<accessKey>`. For more information, see [Basic Access (Base64 encoded)](/docs/api/getting-started#basic-access-base64-encoded).
```

**Review by kimsauce — APPROVED** [review]
Minor thing! Looks good.

File: docs/cloud-soar/automation.md (hunk 1 above)
Comment (jpipkin1) [reply docs/cloud-soar/automation.md:479]: Leaving as-is. That closing character is needed.

**Pull request merged**